Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.18
```

Comment:
While it's good to specify a version, does it need to be this specific? Maybe just `FROM alpine:3`. Not saying it needs to be looser, just asking.

Reply:
I don't know that the Alpine version matters in this case.  3.18.2 is currently the latest version; 3.18 seemed like a safe and stable version to specify.